src/helm/benchmark/presentation/run_specs_decodingtrust.conf

```diff
@@ -0,0 +1,242 @@
+# Main `RunSpec`s for the benchmarking.
+
+entries: [
+  ##### DecodingTrust Trustworthiness Perspectives #####
+
+  ## Toxicity  #####
+  # subject: toxic, non-toxic, gpt-3.5, gpt-4
+
+  {description: "decodingtrust_toxicity_prompts:subject=toxic,model=text", priority=1}
+  {description: "decodingtrust_toxicity_prompts:subject=non-toxic,model=text", priority=1}
+  {description: "decodingtrust_toxicity_prompts:subject=gpt-3.5,model=text", priority=1}
+  {description: "decodingtrust_toxicity_prompts:subject=gpt-4,model=text", priority=1}
+
+  ## Stereotype bias  #####
+  # Tasks: all
+
+  {description: "stereotype_bias:task=all,model=text", priority=1}
```

Comment:
should be `decodingtrust_stereotype_bias:` instead of `stereotype_bias:`

Reply:
Sorry for the oversight. This has been fixed in ca45e9412b61d0b73829e33e3828c007f197c7b2.